forte/data/selector.py

```diff
@@ -37,9 +39,9 @@
 ]
 
 
-class Selector(Generic[InputPackType, OutputPackType]):
-    def __init__(self, **kwargs):
-        pass
+class Selector(Generic[InputPackType, OutputPackType], Configurable):
+    def __init__(self, configs: Optional[Union[Config, Dict[str, Any]]] = None):
```

Comment:
This is not the correct way to use `Configurable`. What we should do is leave the empty `__init__`, and the configurable will handle the configs in `initialize` function.

The only thing we need to do is to call the `initialize` function in `pipeline.initialize` @mylibrar maybe you can add that?

Reply:
Could you clarify this more?
I didn't see an `initialize` method in `Configurable` and seems like right now `Configurable` is using its method `make_configs` to set up configurations.
I thought `Selector.initialize` was only relevant to some states of a selector, which is different than its configs?